test/parser.jl

```diff
@@ -779,6 +782,10 @@ broken_tests = [
     ]
 ]
 
+@testset "Invalid syntax" begin
+    @test !isempty(JuliaSyntax.parse(JuliaSyntax.GreenNode, """[ [],,[] ]""")[2])
```

Comment:
Let's make a more precise test case out of this and add it to the vector of `tests`.

The parser should emit some `error` node and we can test that.